tests/test_ping_serve/test_ping_serve.c

```diff
@@ -26,15 +26,6 @@ static void puts(const char *s)
     solo5_console_write(s, strlen(s));
 }
 
-#define assert(e) do {                              \
```

Comment:
Why is this removed? Is it unused here?

Reply:
Yes, it was not used. It had a platform_exit() call which I could have updated, but as it was not used, I just removed it.